pom.xml

```diff
@@ -16,104 +16,17 @@
 	<description>Federated SQL and XML query engine.</description>
 	<properties>		
 	    <site.url>http://www.jboss.org/teiid</site.url>
-	    <jdk.min.version>1.8</jdk.min.version>
-	    <maven.compiler.source>1.8</maven.compiler.source>
-	    <maven.compiler.target>1.8</maven.compiler.target>
+	    <!--jdk.min.version>1.8</jdk.min.version-->
```

Comment:
Why has this snippet been commented out? Preferably if this is not needed anymore, it should have been removed.

Reply:
create a jira.

Reply:
There already is a JIRA about writing down best practices. We need to make sure, this is one of them.